src/commands/deploy.rs

```diff
@@ -313,15 +313,12 @@ impl DeployCommand {
     async fn check_hippo_healthz(&self) -> Result<()> {
         let hippo_base_url = url::Url::parse(&self.hippo_server_url)?;
         let hippo_healthz_url = hippo_base_url.join("/healthz")?;
-        let result = reqwest::get(hippo_healthz_url.to_string())
-            .await?
-            .error_for_status()?
-            .text()
-            .await?;
-        if result != "Healthy" {
-            return Err(anyhow!("Hippo server {} is unhealthy", hippo_base_url));
+        let status_code = reqwest::get(hippo_healthz_url.to_string()).await?.status();
+
+        if status_code.is_success() {
+            return Ok(());
         }
-        Ok(())
+        Err(anyhow!("Hippo server {} is unhealthy", hippo_base_url))
```

Comment:
nit: this can be simplified by using `bail!(Hippo server {} is unhealthy", hippo_base_url)`.

See [`anyhow::bail`](https://docs.rs/anyhow/1.0.58/anyhow/macro.bail.html) for more details.

Reply:
thanks for pointing me to `bail` will look into it and update the PR